tests/network/vmi_networking.go

```diff
@@ -766,20 +781,32 @@ var _ = SIGDescribe("[Serial][rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com]
 				tcpPort := 8080
 				tests.StartPythonHttpServer(serverVMI, tcpPort)
 
-				assert.XFail("https://github.com/kubevirt/kubevirt/issues/5113", func() {
-					By("Checking ping (IPv6) from vmi to cluster nodes gateway")
-					// Cluster nodes subnet (docker network gateway)
-					// Docker network subnet cidr definition:
-					// https://github.com/kubevirt/project-infra/blob/master/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5
-					Expect(libnet.PingFromVMConsole(serverVMI, "2001:db8:1::1")).To(Succeed())
-				})
-
 				Expect(verifyClientServerConnectivity(clientVMI, serverVMI, tcpPort, k8sv1.IPv6Protocol)).To(Succeed())
 			},
 				table.Entry("with a specific port number [IPv6]", []v1.Port{{Name: "http", Port: 8080}}, ""),
 				table.Entry("without a specific port number [IPv6]", []v1.Port{}, ""),
 				table.Entry("with custom CIDR [IPv6]", []v1.Port{}, "fd10:10:10::/120"),
 			)
+
+			It("[outside_connectivity]should be able to reach the outside world [IPv6]", func() {
+				ipv6Address := "2001:db8:1::1"
+				if flags.IPV4ConnectivityCheckAddress != "" {
+					ipv6Address = flags.IPV4ConnectivityCheckAddress
+				}
+
+				vmi, err := fedoraMasqueradeVMI([]v1.Port{}, "")
+				Expect(err).ToNot(HaveOccurred())
+				vmi, err = virtClient.VirtualMachineInstance(tests.NamespaceTestDefault).Create(vmi)
+				Expect(err).ToNot(HaveOccurred())
+				vmi = tests.WaitUntilVMIReady(vmi, console.LoginToFedora)
+				Expect(configureIpv6(vmi, "")).To(Succeed(), "failed to configure ipv6  on vmi")
+
+				By("Checking ping (IPv6) from vmi to cluster nodes gateway")
+				// Cluster nodes subnet (docker network gateway)
+				// Docker network subnet cidr definition:
+				// https://github.com/kubevirt/project-infra/blob/master/github/ci/shared-deployments/files/docker-daemon-mirror.conf#L5
+				Expect(libnet.PingFromVMConsole(vmi, ipv6Address)).To(Succeed())
```

Comment:
you removed the XFAIL , we are fine with it right?

Reply:
according to [this](https://github.com/kubevirt/kubevirt/issues/5113#issuecomment-814238968) we are. Don't we?

Reply:
if all is stable, sure

Reply:
It is also says so in the commit message, but perhaps it is not clear enough . I'll rephrase it.